frontend/src/components/admin/ComplaintManagement.jsx

```
@@ -33,53 +45,69 @@ const ComplaintManagement = () => {
 
       // 응답 데이터 처리
+      let complaintData = [];
       if (response.data) {
-        if (response.data.content && Array.isArray(response.data.content)) {
-          // 페이징된 응답 처리
-          setComplaints(response.data.content);
+        if (response.data.data && response.data.data.content && Array.isArray(response.data.data.content)) {
+          // 페이징된 응답 처리 - 중첩된 구조
+          complaintData = response.data.data.content;
+          console.log('✅ response.data.data.content 경로 사용');
+          setPagination(prev => ({
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)